> I the original version, there was a leading / Now it is a long line issue. That is the disadvantage if our two stage clang-format call.
> 
> Do you have pre-commit installed, I wonder why this has slipped through.

precommit is broken on my machine and I have been unable to fix it:

```An unexpected error has occurred: AssertionError: BUG: expected environment for python to be healthy immediately after install, please open an issue describing your environment

more info:

virtualenv python version did not match created version:
- actual version: <<error retrieving version from /home/owen/.cache/pre-commit/repo2_q5mxbt/py_env-python3/bin/python>>
- expected version: 3.10.6.final.0
```

--- a/src/engine/enginebuffer.cpp
+++ b/src/engine/enginebuffer.cpp
@@ -1274,8 +1274,8 @@ void EngineBuffer::updateIndicators(double speed, int iBufferSize) {
 
     const double fFractionalPlaypos = fractionalPlayposFromAbsolute(m_filepos_play);
 
-    const double tempoTrackSeconds = m_trackSamplesOld / kSamplesPerFrame
-            / m_trackSampleRateOld / m_tempo_ratio_old;
+    const double tempoTrackSeconds = m_trackSamplesOld / kSamplesPerFrame /
+            m_trackSampleRateOld / getRateRatio();
     if (speed > 0 && fFractionalPlaypos == 1.0) {
         // At Track end
         speed = 0;

